examples/imatrix/imatrix.cpp

```diff
@@ -112,6 +112,7 @@ bool IMatrixCollector::collect_imatrix(struct ggml_tensor * t, bool ask, void *
         // this is necessary to guarantee equal number of "ncall" for each tensor
         for (int ex = 0; ex < n_as; ++ex) {
             src0 = t->src[2 + ex];
+            wname = src0->name;
```

Comment:
The name should also be filtered here, it may contain the sched affixes with partial offload.